browser/ui/toolbar/app_menu_icons.cc

```diff
@@ -32,7 +32,6 @@ const std::map<int, const gfx::VectorIcon&>& GetCommandIcons() {
           {IDC_SHOW_BRAVE_REWARDS, kLeoProductBatOutlineIcon},
           {IDC_SHOW_BRAVE_WALLET, kLeoProductBraveWalletIcon},
           {IDC_BRAVE_VPN_MENU, kLeoProductVpnIcon},
-          {IDC_APP_MENU_IPFS, kLeoProductIpfsOutlineIcon},
```

Comment:
It seems we don't need `leo_product_ipfs_outline.icon` in `brave/components/vector_icons/BUILD.GN`